test/test.js

```diff
@@ -1,13 +1,26 @@
+"use strict";
```

Comment:
it's for `node 0.10.0 / 0.12.0`. When I try to fix tests for this versions. It allow to pass, but it stuck on using dependencies